src/transport/freebsd/device.rs

```diff
@@ -57,7 +59,10 @@ impl Device {
             }
 
             // Read response
-            self.read_exact(&mut buf)?;
+            let n = self.read(&mut buf[1..])?;
```

Comment:
Might be nice to have a comment like in #234 explaining why we don't need to read the report number here.